qemu/tests/pci_hotplug.py

```diff
@@ -53,10 +53,10 @@ def pci_add(pci_add_cmd):
 
     def is_supported_device(dev):
         # Probe qemu to verify what is the supported syntax for PCI hotplug
-        cmd_output = vm.monitor.human_monitor_cmd("?")
-        if len(re.findall("\ndevice_add", cmd_output)) > 0:
+        vm.monitor._get_supported_cmds()
+        if vm.monitor._has_command("device_add"):
```

Comment:
functions start with _ in python is kind of 'internal only' functions not suggested to used outside.
Please reference this: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles
item: _single_leading_underscore.